### PR TITLE
fix: refresh welcome page - steer towards OIDC bridge

### PR DESCRIPTION
**Pull request by vladimir-mencl-eresearch**
Change messaging to steer users towards Tuakiri OopenID Connect Bridge.

Remove links to dead example applications.

### CODE REVIEW

File: rapidconnect/app/views/welcome.erb [hunk 1]
```diff
@@ -28,13 +28,17 @@
...
+            <h3 class="text-muted">About Rapid Connect</h3>
             <ul>
+
+              <li>Rapid Connect provides an easy way for applicaitons to authenticate users via a signed Json Web Token (JWT)</li>
               <li>There is <strong>no need to install a Shibboleth SP</strong> on your webserver;</li>
               <li>It natively integrates into commonly used development languages;</li>
               <li>PaaS solutions like Heroku, Google App Engine and Pagoda become suitable deployment targets for Tuakiri services;</li>
               <li>Attributes are already defined in logical sets, there is no approval process for attributes; and</li>
               <li>Integration code is <strong>minimal</strong>, simple to write and <strong>easy to test</strong>.</li>
+              <li>However, JWT and Rapid Connect originate from before the <a href="https://openid.net/developers/how-connect-works/">OpenID Connect specification</a> was created and adtopted.</li>
```
Comment (James-REANNZ) [inline rapidconnect/app/views/welcome.erb:40]: Typo!

```suggestion
              <li>However, JWT and Rapid Connect originate from before the <a href="https://openid.net/developers/how-connect-works/">OpenID Connect specification</a> was created and adopted.</li>
```

File: rapidconnect/app/views/welcome.erb [hunk 2]
```diff
@@ -28,13 +28,17 @@
       <div class="container">
         <div class="row">
           <div class="col-md-6">
-            <h3 class="text-muted">Benefits</h3>
+            <h3 class="text-muted">About Rapid Connect</h3>
             <ul>
+
+              <li>Rapid Connect provides an easy way for applicaitons to authenticate users via a signed Json Web Token (JWT)</li>
```
Comment (James-REANNZ) [inline rapidconnect/app/views/welcome.erb:34]:
```suggestion
              <li>Rapid Connect provides an easy way for applications to authenticate users via a signed Json Web Token (JWT)</li>
```

**Comment by James-REANNZ** [comment]
Other than the two spelling errors, looks good.

**Comment by vladimir-mencl-eresearch** [comment]
Thanks for the quick review - typos fixed (and force-pushed for shorter commit history).

**Pull request merged**